packages/lpa-questionnaire-web-app/src/controllers/accuracy-submission.js

```diff
@@ -7,3 +7,26 @@ exports.getAccuracySubmission = (req, res) => {
     backLink: req.session.backLink || `/${req.params.id}/${VIEW.TASK_LIST}`,
   });
 };
+
+exports.postAccuracySubmission = async (req, res) => {
+  const { body } = req;
+  const { errors = {}, errorSummary = [] } = body;
```

Comment:
errors and errorSummary used once. Do we need a variable?

Reply:
They get used more later, so it's sort of planning ahead, knowing what the final version looks like